.github/workflows/bindings.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - name: Build bindings
         run: |
           sudo apt-get install -y node-gyp
-          cd $GITHUB_WORKSPACE/ffi/nodejs
+          cd $GITHUB_WORKSPACE/ffi/npm
```

Comment:
I don't understand this name change, npm is a package manger, not a language

Moreover I wanted to propose to change it to a more general js (which internally can build nodejs or v8 bindings, maybe providing to the user a flag). Same for iOS (which can be Swift or iOS) and for Android (which can be Java or Android)

Reply:
Let's discuss this on the call, I will explain the rational behind the names and the new proposed structure